react/src/RestrictedFooter/RestrictedFullFooter.tsx

```diff
@@ -85,7 +86,10 @@ export const RestrictedFullFooter = ({
             mb="2rem"
             href={footerIconLink.href}
           >
-            <footerIconLink.Icon width="183px"></footerIconLink.Icon>
+            <footerIconLink.Icon
+              textFill={logoTextColor}
+              width="183px"
+            ></footerIconLink.Icon>
```

Comment:
i suspect this is also not needed :P. Users of the footer can override the footer link icon with a custom one:

```tsx
// custom footer icon link in user's app
const customFooterLink: FooterLinkWithIcon = {
  href: 'https://open.gov.sg',
  label: 'Open Government Products homepage',
  Icon: CustomLogo, // this can be the logo that is correctly filled with the new colors to fit monochrome footers
}

// in the footer
<RestrictedFullFooter
  ... 
  footerIconLink={customFooterLink}
/>
```

Reply:
redid the implementation to accommodate different logo variants